app/controllers/api/v2/environments_controller.rb

```diff
@@ -61,6 +61,13 @@ def destroy
       def allowed_nested_id
         %w(puppetclass_id location_id organization_id)
       end
+
+      def find_resource
```

Comment:
Removed friendly_id because users could not use environment names with friendly_id reserved names, e.g., If you have an environment named "new", you could not import it